docs/en-us/dev/user_doc/registry_spi.md

```diff
@@ -0,0 +1,26 @@
+### DolphinScheduler Registry SPI main design
+
+#### How to use it？
+
+First you need to execute the `mvn -U install package -Prelease -Dmaven.test.skip=true` to install the plugin for generating the plugin JAR of the registry. The directory is: dolphinscheduler-dist/target/dolphinscheduler-dist-1.3.6-SNAPSHOT/lib/plugin/registry.
+
+Next, do the following configuration (using zookeeper as an example)
+
+|        Parameter        | Default                                                      | Description                              |
+| :---------------------: | ------------------------------------------------------------ | ---------------------------------------- |
+|   registry.plugin.dir   | /Users/username/workspaces/dolphinscheduler/dolphinscheduler-dist/target/dolphinscheduler-dist-1.3.6-SNAPSHOT/lib/plugin/registry | Registration Center Plugin Directory     |
+| registry.plugin.name\|  | zookeeper                                                    | Registration Center specific plugin name |
```

Comment:
\ is redundant here.

Another thing is that only the first letters of the words in the first row of the Description column are capitalized. You should keep the consistency like changing the first letters of the remaining rows to upper case or changing the first letters of the first row to lower case.